tests/components/scrubberBar-test.js

```diff
@@ -58,6 +99,80 @@ describe('ScrubberBar', function () {
     expect(ReactDOM.findDOMNode(DOM.refs.playhead).style.backgroundColor).toBe("blue");
   });
 
+  it('should render ARIA attributes', function () {
+    baseMockController.state.currentPlayhead = 0;
+    baseMockController.state.duration = 60;
+    updateBaseMockProps();
+    var DOM = TestUtils.renderIntoDocument(<ScrubberBar {...baseMockProps}/>);
+    var scrubberBar = TestUtils.findRenderedDOMComponentWithClass(DOM, "oo-scrubber-bar");
+    expect(scrubberBar.getAttribute('aria-label')).toBe(CONSTANTS.ARIA_LABELS.SEEK_SLIDER);
+    expect(scrubberBar.getAttribute('aria-valuemin')).toBe("0");
+    expect(scrubberBar.getAttribute('aria-valuemax')).toBe(baseMockController.state.duration.toString());
+    expect(scrubberBar.getAttribute('aria-valuenow')).toBe(baseMockController.state.currentPlayhead.toFixed(2));
+    expect(scrubberBar.getAttribute('aria-valuetext')).toBe('00:00 of 01:00');
+    expect(scrubberBar.getAttribute('data-focus-id')).toBe('scrubberBar');
+  });
+
+  it('should update the ARIA value in order to reflect the current playhead', function() {
+    baseMockController.state.currentPlayhead = 2;
+    baseMockController.state.duration = 60;
+    updateBaseMockProps();
+    var DOM = TestUtils.renderIntoDocument(<ScrubberBar {...baseMockProps}/>);
+    var scrubberBar = TestUtils.findRenderedDOMComponentWithClass(DOM, "oo-scrubber-bar");
+    expect(scrubberBar.getAttribute('aria-valuenow')).toBe('2.00');
+    expect(scrubberBar.getAttribute('aria-valuetext')).toBe('00:02 of 01:00');
+
+    baseMockController.state.currentPlayhead = 60;
```

Comment:
What happens when currentPlayhead is greater than duration with regards to the aria attributes? I know we have safeguards to prevent this from happening, but if it does happen, would anything break?

Reply:
The screen reader might read out the wrong value

Reply:
That's fine.